src/test/kotlin/blackjack/domain/card/EmptyCardFactory.kt

```diff
@@ -0,0 +1,9 @@
+package blackjack.domain.card
+
+import blackjack.domain.Card
+
+class EmptyCardFactory : CardFactory {
+    override fun createCards(): List<Card> {
+        return mutableListOf()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        return emptyList()
```